extensions/reactive-datasource/runtime/src/main/java/io/quarkus/reactive/datasource/runtime/DataSourceReactiveRuntimeConfig.java

```diff
@@ -23,8 +23,8 @@ public interface DataSourceReactiveRuntimeConfig {
     /**
      * Whether prepared statements should be cached on the client side.
      */
-    @WithDefault("false")
-    boolean cachePreparedStatements();
+    @ConfigDocDefault("true for PostgreSQL/MySQL/MariaDB/DB2/Oracle, false otherwise")
```

Comment:
```suggestion
    @ConfigDocDefault("true for PostgreSQL/MySQL/MariaDB/DB2, false otherwise")
```

In fact, it's not supported in the Reactive Oracle client